api/src/main/resources/workbench-api.yaml

```diff
@@ -2423,11 +2401,33 @@ paths:
           "$ref": "#/definitions/Institution"
       responses:
         200:
-          description: 'The updated Institution.
-
-'
+          description: The updated Institution.
           schema:
             "$ref": "#/definitions/Institution"
+  "/v1/institutions/{shortName}/checkEmail/{email}/":
```

Comment:
Today I learned: the trailing slash here is important. I'd initially left it out, and what happened was that an API call such as:

/v1/institutions/VUMC/checkEmail/gjuggler%40gmail.com

would land in our controller with parameters like:

shortName: VUMC
email: gjuggler@gmail

without the trailing ".com".

Adding the trailing slash to the Swagger helped fix this – the client now sends requests with the trailing slash, and the controller parses them correctly.

(Note to self: if we were truly paranoid here, we'd add some more extensive controller-level tests to exercise the full request parsing flow. I didn't think it was worth it in this case, but I'm happy to hear disagreement.)